blackjax/adaptation/meads.py

```diff
@@ -26,10 +26,7 @@ class MEADSAdaptationState(NamedTuple):
     delta: float
 
 
-def base(
-    logprob_grad_fn: Callable,
-    batch_fn: Callable = jax.vmap,
-):
+def base():
```

Comment:
Seems unnecessary - should we just remove this and let the function exists in the namespace?